src/issue/screens/issue.screen.js

```diff
@@ -254,12 +283,19 @@ class Issue extends Component {
       isPendingComments,
       isPendingContributors,
       isPendingIssue,
+      isDeletingComment,
       language,
       navigation,
     } = this.props;
 
-    const isLoadingData = !!(isPendingComments || isPendingIssue);
+    const isLoadingData = !!(
+      isPendingComments ||
+      isPendingIssue ||
+      isDeletingComment
+    );
+    const showLoadingContainer = isPendingComments || isPendingIssue;
```

Comment:
Since this variable contains a boolean value, it can be renamed to `isShowLoadingContainer`?